app/src/main/res/values-en/strings.xml

```diff
@@ -113,34 +113,34 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="at_housename">house name %s</string>
     <string name="at_housenumber">house number %s</string>
     <!-- when the user left the answer form empty -->
-    <string name="no_changes">"You did not give any answer"</string>
```

Comment:
Note that it appears when user gave a partial answer.

Reply:
So "you did not give a complete answer" then?

Reply:
I went with "did not give an answer" based on the "when the user left the answer form empty" comment above it so maybe that needs to be updated also

Reply:
wait, that is a different thing please disregard